portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Entity;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Models a servlet for deleting data from the datastore
+ */
+@WebServlet("/delete-data")
+public class DeleteDataServlet extends HttpServlet {
+
+  /**
+   * Deletes the contents of the datastore.
+   * @param request the server request
```

Comment:
I'm thinking you don't necessarily have to document the parameters if theres nothing much to say about them that isn't already covered by their name.